specification/recoveryservicessiterecovery/resource-manager/Microsoft.RecoveryServices/stable/2021-02-10/service.json

```diff
@@ -17223,8 +20032,42 @@
       },
       "x-ms-discriminator-value": "InMage"
     },
+    "RecoveryPlanInMageRcmFailbackFailoverInput": {
+      "description": "Recovery plan InMageRcmFailback failover input.",
+      "required": [
+        "recoveryPointType"
+      ],
+      "type": "object",
+      "allOf": [
+        {
+          "$ref": "#/definitions/RecoveryPlanProviderSpecificFailoverInput"
+        }
+      ],
+      "properties": {
+        "recoveryPointType": {
+          "description": "The recovery point type. See SrsDataContract.RecoveryPoint enum for possible values.",
```

Comment:
Remove - _"See SrsDataContract.RecoveryPoint enum for possible values."_

Reply:
removed

---
In reply to: [588096813](https://github.com/AsrOneSdk/azure-rest-api-specs/pull/16#discussion_r588096813) [](ancestors = 588096813)